src/test/ratings.spec.js

```diff
@@ -0,0 +1,51 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+
+chai.use(chaiHttp);
+chai.should();
+
+describe('Testing Rating of articles', () => {
+  it('it should return a 400 status code if the request is bad', done => {
+    chai
+      .request(app)
+      .patch('/api/v1/articles/:articleId/:ratings')
+      .send({})
+      .end((error, response) => {
+        expect(response.status).to.equal(400);
+        done();
+      });
+  });
+  it('it should return an relevant error message if the request is bad', done => {
+    chai
+      .request(app)
+      .patch(`/api/v1/articles/:articles/:ratings`)
+      .end((error, response) => {
+        response.body.message.should.be.a('string');
```

Comment:
This test doesn't demonstrate that a relevant error message was sent. An empty string is a string. What is the message. We should know the exact message that was sent.